docs/content/docs/projects/BuildChecks.md

```diff
@@ -48,18 +48,49 @@ A problem occurred configuring root project
 
 After adding missing properties you will get something like this:
 
+{{< tabs "build checks example" >}}
```

Comment:
Configs in Kotlin and Groovy are the same. But it's good to have both variants for the sake of consistent documentation.